lrpar/cttests/src/cgen_helper.rs

```diff
@@ -79,10 +79,10 @@ pub(crate) fn run_test_path<P: AsRef<Path>>(path: P) -> Result<(), Box<dyn std::
         // Create grammar files
         let base = path.file_stem().unwrap().to_str().unwrap();
         let mut pg = PathBuf::from(&out_dir);
-        pg.push(format!("{}.y.rs", base));
+        pg.push(format!("{}.test.y", base));
```

Comment:
I had believed that this `.y.rs` thing was a mistake of sorts, because this is a `.y` file not a `.rs` file.
But it occurred to me that emitting of this file with a .rs extension may have been a way to get the same behavior as `cargo:rerun-if-changed=path` behavior.

In which case I don't think it's necessary we emit `rerun-if-changed` for the .test file these derive from.
But I do think that means we should also emit rerun-if-changed for `glob(src/*.rs)` which I *don't* think we're doing.

Perhaps best to do that and an audit in a follow up.

Reply:
> Perhaps best to do that and an audit in a follow up.

Good idea!